### ISSUE TITLE
loss value stays constant during the training

### ISSUE THREAD

**Issue by HariKrishna-Vydana** [mcf06/theano_ctc#16]

**Comment by githubnemo** [comment]
I have not experienced this and the mini batch size is passed to warp-ctc. Can you give a minimal example demonstrating this effect?

**Comment by HariKrishna-Vydana** [comment]
@githubnemo i am using a minibatch size of 10,running on keras
the error is drastcically high around 1000,which is not the case for batch_size of 1

**Comment by githubnemo** [comment]
OK but can you share your code that reproduces this behavior?
